tox.ini

```diff
@@ -80,7 +80,9 @@ commands =
 
 [testenv:pylint]
 basepython = {env:BASEPYTHON:py312}
-deps = pylint
+deps =
+    pylint
+    .[tests]
```

Comment:
(This one fixes inability to reproduce the error when running lint locally via tox)